src/main/java/org/interledger/InterledgerAddress.java

```diff
@@ -271,7 +275,7 @@ public int hashCode() {
 
       @Override
       public String toString() {
-        return "InterledgerAddress.Impl{value='" + value + "'}";
+        return this.getValue();
```

Comment:
Any objections to this change? I think I advocated (in the past) for the opposite of what I'm about to say, but for `InterledgerAddress`, simply emitting the `value` as the output of `toString` seems like what I would expect as a developer. Another few arguments in favor of this change:

* Seeing the JSON version of itself doesn't seem terribly helpful in this case since there's only 1 field -- the "value".
* If we emit `value` when calling toString, it's generally pretty obvious that this is an `InterledgerAddress` due to the nature of the format of an InterledgerAddress (so, no need to overtly state that in a JSON payload with one value).